src/python/pants/backend/python/target_types.py

```diff
@@ -280,29 +280,13 @@ class PythonBinary(Target):
     )
 
 
-# -----------------------------------------------------------------------------------------------
-# `python_library` target
-# -----------------------------------------------------------------------------------------------
-
-
-class PythonLibrarySources(PythonSources):
-    default = ("*.py", "!test_*.py", "!*_test.py", "!conftest.py")
-
-
-class PythonLibrary(Target):
-    """A Python library that may be imported by other targets."""
-
-    alias = "python_library"
-    core_fields = (*COMMON_PYTHON_FIELDS, PythonLibrarySources)
-
-
 # -----------------------------------------------------------------------------------------------
 # `python_tests` target
 # -----------------------------------------------------------------------------------------------
 
 
 class PythonTestsSources(PythonSources):
-    default = ("test_*.py", "*_test.py", "conftest.py")
+    default = ("tests.py", "test_*.py", "*_test.py", "conftest.py")
```

Comment:
Nit: I recommend putting `tests.py` at the end of the list. It makes no difference to Pants, but impacts what shows up in `./pants target-types --details=python_tests`. The `*` globs are more general so seem to be better in the front.